### PR TITLE
Try: Escape apostrophe in translation functions.

### PR DESCRIPTION
**Pull request by juanfra**
**Description**

Replacing `&rsquo;` for an escaped apostrophe (`\'`) in translatable texts.

**Screenshots**

- N/A

**Testing Instructions**

1. Create a page.
2. Insert all the patterns modified in this PR.
3. Confirm the texts are looking as they were before.

### CODE REVIEW

**Comment by carolinan** [comment]
Why, what is the benefit of this change?

**Comment by juanfra** [comment]
> Why, what is the benefit of this change?

From my experience working with translations, I see that many translators are not technical and don't know about these entities. The benefit would be that it'll be easier for translators to do their work.

I believe historically, in core (and other products I've been involved with), the tendency is to use apostrophes rather than entities.

**Comment by carolinan** [comment]
That sounds very reasonable to me.

It does not match how these symbols have been managed in the previous bundled themes.
I would like to ask @SergeyBiryukov but he is on vacation until next week.

**Comment by carolinan** [comment]
I also recognize that on trunk there seems to be a mix of slashes and entities which was not intended :)

File: patterns/banner-about-book.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
+			<p class="has-medium-font-size"><?php echo esc_html_x( 'This exquisite compilation showcases a diverse array of photographs that capture the essence of different eras and cultures, reflecting the unique styles and perspectives of each artist. Fleckenstein\'s evocative imagery, Strand\'s groundbreaking modernist approach, and Kōno\'s meticulous documentation of Japanese life come together in a harmonious blend that celebrates the art of photography. Each image in “The Stories Book” is accompanied by insightful commentary, providing historical context and revealing the stories behind the photographs. This collection is not only a visual feast but also a tribute to the power of photography to preserve and narrate the multifaceted experiences of humanity.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```
Comment (SergeyBiryukov) [inline patterns/banner-about-book.php]:
```suggestion
			<p class="has-medium-font-size"><?php echo esc_html_x( 'This exquisite compilation showcases a diverse array of photographs that capture the essence of different eras and cultures, reflecting the unique styles and perspectives of each artist. Fleckenstein’s evocative imagery, Strand’s groundbreaking modernist approach, and Kōno’s meticulous documentation of Japanese life come together in a harmonious blend that celebrates the art of photography. Each image in “The Stories Book” is accompanied by insightful commentary, providing historical context and revealing the stories behind the photographs. This collection is not only a visual feast but also a tribute to the power of photography to preserve and narrate the multifaceted experiences of humanity.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```

File: patterns/banner-poster.php [hunk 2]
```diff
@@ -41,7 +41,7 @@
 			<!-- wp:column {"verticalAlignment":"bottom","width":"66.66%"} -->
 			<div class="wp-block-column is-vertically-aligned-bottom" style="flex-basis:66.66%">
 				<!-- wp:heading {"textAlign":"left","align":"wide","style":{"typography":{"lineHeight":"0.9","fontStyle":"normal","fontWeight":"300"}},"fontSize":"xx-large"} -->
-				<h2 class="wp-block-heading alignwide has-text-align-left has-xx-large-font-size" style="font-style:normal;font-weight:300;line-height:0.9"><?php esc_html_e( 'Let&rsquo;s hear them.', 'twentytwentyfive' ); ?></h2>
+				<h2 class="wp-block-heading alignwide has-text-align-left has-xx-large-font-size" style="font-style:normal;font-weight:300;line-height:0.9"><?php esc_html_e( 'Let\'s hear them.', 'twentytwentyfive' ); ?></h2>
```
Comment (SergeyBiryukov) [inline patterns/banner-poster.php]:
```suggestion
				<h2 class="wp-block-heading alignwide has-text-align-left has-xx-large-font-size" style="font-style:normal;font-weight:300;line-height:0.9"><?php esc_html_e( 'Let’s hear them.', 'twentytwentyfive' ); ?></h2>
```

File: patterns/event-schedule.php [hunk 3]
```diff
@@ -86,7 +86,7 @@
 								<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
 								<div class="wp-block-group">
 									<!-- wp:heading {"level":4} -->
-									<h4 class="wp-block-heading"><a href="#"><?php esc_html_e( 'Things you didn&rsquo;t know about the deep ocean', 'twentytwentyfive' ); ?></a></h4>
+									<h4 class="wp-block-heading"><a href="#"><?php esc_html_e( 'Things you didn\'t know about the deep ocean', 'twentytwentyfive' ); ?></a></h4>
```
Comment (SergeyBiryukov) [inline patterns/event-schedule.php]:
```suggestion
									<h4 class="wp-block-heading"><a href="#"><?php esc_html_e( 'Things you didn’t know about the deep ocean', 'twentytwentyfive' ); ?></a></h4>
```

File: patterns/page-cv-bio.php [hunk 4]
```diff
@@ -29,7 +29,7 @@
...
+					<p class="is-style-text-subtitle"><?php echo esc_html_x( 'My name is Nora Winslow Keene, and I\'m a committed public interest attorney. Living in Denver, Colorado, I\'ve spent years championing the rights of underrepresented workers. A graduate of Stanford University, I played a key role in securing critical protections for agricultural laborers, ensuring better wages and access to healthcare. My work has focused on advocating for environmental justice and improving the quality of life for rural communities. Every case I take on is driven by the belief that everyone deserves dignity and fair treatment in the workplace.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```
Comment (SergeyBiryukov) [inline patterns/page-cv-bio.php]:
```suggestion
					<p class="is-style-text-subtitle"><?php echo esc_html_x( 'My name is Nora Winslow Keene, and I’m a committed public interest attorney. Living in Denver, Colorado, I’ve spent years championing the rights of underrepresented workers. A graduate of Stanford University, I played a key role in securing critical protections for agricultural laborers, ensuring better wages and access to healthcare. My work has focused on advocating for environmental justice and improving the quality of life for rural communities. Every case I take on is driven by the belief that everyone deserves dignity and fair treatment in the workplace.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```

File: patterns/page-link-in-bio-wide-margins.php [hunk 5]
```diff
@@ -35,7 +35,7 @@
 				<!-- /wp:heading -->
 
 				<!-- wp:paragraph -->
-				<p><?php echo esc_html_x( 'I&rsquo;m Nora, a dedicated public interest attorney based in Denver. I&rsquo;m a graduate of Stanford University.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
+				<p><?php echo esc_html_x( 'I\'m Nora, a dedicated public interest attorney based in Denver. I\'m a graduate of Stanford University.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```
Comment (SergeyBiryukov) [inline patterns/page-link-in-bio-wide-margins.php]:
```suggestion
				<p><?php echo esc_html_x( 'I’m Nora, a dedicated public interest attorney based in Denver. I’m a graduate of Stanford University.', 'Pattern placeholder text.', 'twentytwentyfive' ); ?></p>
```

File: patterns/page-link-in-bio-with-tight-margins.php [hunk 6]
```diff
@@ -39,7 +39,7 @@
 			<!-- wp:group {"style":{"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","flexWrap":"nowrap","justifyContent":"stretch"}} -->
 			<div class="wp-block-group" style="min-height:100%">
 				<!-- wp:paragraph {"align":"left","style":{"typography":{"lineHeight":"1.2"}},"fontSize":"x-large"} -->
-				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I&rsquo;m Asahachi Kōno, a Japanese&nbsp;photographer, a member of&nbsp;Los Angeles\'s Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
+				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I\'m Asahachi Kōno, a Japanese photographer, a member of Los Angeles\'s Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
```
Comment (SergeyBiryukov) [inline patterns/page-link-in-bio-with-tight-margins.php]:
```suggestion
				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I’m Asahachi Kōno, a Japanese photographer, a member of Los Angeles’ Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
```

**Review by SergeyBiryukov — CHANGES_REQUESTED** [review]
Thanks for the PR! I would say these are fine to replace, and I see that some of the previous bundled themes already use the actual double quote characters instead of entities.

However, entities are generally used for better typography, so instead of the escaped apostrophe character, I would suggest using an actual single quote, which would not need escaping and also has a precedent in the previous bundled themes.

Left some suggestions, hope this helps 🙂

**Comment by juanfra** [comment]
Thank you both!

File: patterns/page-link-in-bio-with-tight-margins.php [hunk 7]
```diff
@@ -39,7 +39,7 @@
 			<!-- wp:group {"style":{"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","flexWrap":"nowrap","justifyContent":"stretch"}} -->
 			<div class="wp-block-group" style="min-height:100%">
 				<!-- wp:paragraph {"align":"left","style":{"typography":{"lineHeight":"1.2"}},"fontSize":"x-large"} -->
-				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I&rsquo;m Asahachi Kōno, a Japanese&nbsp;photographer, a member of&nbsp;Los Angeles\'s Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
+				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I’m Asahachi Kōno, a Japanese photographer, a member of Los Angeles’ Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
```
Comment (juanfra) [inline patterns/page-link-in-bio-with-tight-margins.php]:
```suggestion
				<p class="has-text-align-left has-x-large-font-size" style="line-height:1.2"><?php esc_html_e( 'I’m Asahachi Kōno, a Japanese photographer, a member of Los Angeles’s Japanese Camera Pictorialists of California. Before returning to Japan, I worked as a photo retoucher.', 'twentytwentyfive' ); ?></p>
```

**Pull request merged**